src/handlers/ecommerce/transaction.ts

```diff
@@ -1432,148 +1478,232 @@ export const ecommerceGetTransaction: RequestHandler = async (req, res) => {
+      );
     case FlowCase.CLOSURE_ERROR_WITH_NPG_AUTH_STATUS_DENIED_BY_RISK:
-      return res
-        .status(200)
-        .send(
-          createSuccessGetTransactionEntity(
-            req.params.transactionId,
-            TransactionStatusEnum.CLOSURE_ERROR,
-            NPG_GATEWAY,
-            undefined,
-            undefined,
-            NpgAuthorizationStatus.DENIED_BY_RISK
-          )
-        );
+      return res.status(200).send(
+        createSuccessGetTransactionEntity(
+          req.params.transactionId,
+          TransactionStatusEnum.CLOSURE_ERROR,
+          {
+            sendPaymentResultOutcome: undefined
+          },
+          {
+            authorizationStatus: NpgAuthorizationStatus.DENIED_BY_RISK,
+            gateway: NPG_GATEWAY
+          }
+        )
+      );
     case FlowCase.CLOSURE_ERROR_WITH_NPG_AUTH_STATUS_THREEDS_VALIDATED:
-      return res
-        .status(200)
-        .send(
-          createSuccessGetTransactionEntity(
-            req.params.transactionId,
-            TransactionStatusEnum.CLOSURE_ERROR,
-            NPG_GATEWAY,
-            undefined,
-            undefined,
-            NpgAuthorizationStatus.THREEDS_VALIDATED
-          )
-        );
+      return res.status(200).send(
+        createSuccessGetTransactionEntity(
+          req.params.transactionId,
+          TransactionStatusEnum.CLOSURE_ERROR,
+          {
+            sendPaymentResultOutcome: undefined
+          },
+          {
+            authorizationStatus: NpgAuthorizationStatus.THREEDS_VALIDATED,
+            gateway: NPG_GATEWAY
+          }
+        )
+      );
     case FlowCase.CLOSURE_ERROR_WITH_NPG_AUTH_STATUS_THREEDS_FAILED:
-      return res
-        .status(200)
-        .send(
-          createSuccessGetTransactionEntity(
-            req.params.transactionId,
-            TransactionStatusEnum.CLOSURE_ERROR,
-            NPG_GATEWAY,
-            undefined,
-            undefined,
-            NpgAuthorizationStatus.THREEDS_FAILED
-          )
-        );
+      return res.status(200).send(
+        createSuccessGetTransactionEntity(
+          req.params.transactionId,
+          TransactionStatusEnum.CLOSURE_ERROR,
+          {
+            sendPaymentResultOutcome: undefined
+          },
+          {
+            authorizationStatus: NpgAuthorizationStatus.THREEDS_FAILED,
+            gateway: NPG_GATEWAY
+          }
+        )
+      );
+    case FlowCase.CLOSURE_ERROR_WITH_NPG_ON_CLOSE_PAYMENT_ERROR_CODE_422_DID_NOT_RECEIVE_RPT:
+      return res.status(200).send(
+        createSuccessGetTransactionEntity(
+          req.params.transactionId,
+          TransactionStatusEnum.CLOSURE_ERROR,
+          {
+            closePaymentResultError: {
+              description: "Node did not receive RPT yet",
+              statusCode: 422
+            },
+            sendPaymentResultOutcome: undefined
+          },
+          {
+            authorizationStatus: NpgAuthorizationStatus.DECLINED,
```

Comment:
```suggestion
            authorizationStatus: NpgAuthorizationStatus.EXECUTED,
```
See comment on ecommerce-fe and checkout prs

Reply:
done here 7a15fede665b37e769eac0fdde94a02f8b0e8f91